### ISSUE TITLE
Paste text keyword issue with updated SikuliLibrary

### ISSUE THREAD

**Issue by Marnok** [rainmanwy/robotframework-SikuliLibrary#57]
Hi,

Paste text keyword doesn't seem to work properly with new updated SikuliLibrary (version 1.0.1 version.py).

On older versions calling keyword from robot `Paste Text        \        cmd` worked, now same results with error:
![pastetextfails](https://user-images.githubusercontent.com/5380923/33836214-76740656-de91-11e7-97ad-9b5ad64eb257.PNG)

I'm running SikuliLibrary server applet on a remote device running Windows 10 and have configured sikuli.py to use Python mode. EDIT: This is exactly same setup which were used with older SikuliLibrary

Input text seems to work fine with same parameters as above.

**Comment by rainmanwy** [comment]
@Marnok , this keyword will call "paste" method of sikulix.


int | paste(java.lang.String text)pastes the text at the current position of the focus/carret using the clipboard and strg/ctrl/cmd-v (paste keyboard shortcut)
-- | --

Could you confirm that the position of focuse is correct when doing "paste text"

**Comment by Marnok** [comment]
@rainmanwy Hi, yes the focus is correct. I changed test so it will definitely will be focused correctly on search bar. Paste text resulted with same error again.

**Comment by rainmanwy** [comment]
@Marnok , could you attach the .log file here. the console output of sikuli java process will be write to log file(not rf log.html).

by the way, recently this lib upgraded to use sikulixapi 1.1.1, So the old version work fine, but with the latest version, "paste text" not work well, am i correct?

**Comment by Marnok** [comment]
Hi @rainmanwy , where sikuli java process writes those .log logs by default?

**Comment by rainmanwy** [comment]
@Marnok ,

`    def _output_file(self):
        outputDir = self._get_output_folder()
        outputFile = 'Sikuli_java_stdout_' + str(time.time()) + '.txt'
        return os.path.join(outputDir, outputFile)

    def _err_file(self):
        outputDir = self._get_output_folder()
        errFile = 'Sikuli_java_stderr_' + str(time.time()) + '.txt'
        return os.path.join(outputDir, errFile)
`

and these two log files should be in RF output dir folder, that mean it should be in same folder with log.html/output.xml/report.html

**Comment by Eireen2011** [comment]
@rainmanwy 
I had the same issue, SikuliLibrary 1.0.1

10:37:21.570 | INFO | Params: [, tHHI1806]
10:37:21.570 | INFO | Params: [, tHHI1806] Paste Text: tHHI1806 |  
10:37:21.570 | FAIL | com.github.rainmanwy.robotframework.sikulilib.exceptions.ScreenOperationException: Paste text failed


Log is [RIDE5h7ixi.d.zip](https://github.com/rainmanwy/robotframework-SikuliLibrary/files/1892743/RIDE5h7ixi.d.zip)

**Comment by rainmanwy** [comment]
Thank you all.
I found the return value from sikuli paste is changed.
I have updated the code, you may get the latest code from github.

**Comment by Marnok** [comment]
Thanks for fixing this issue @rainmanwy . I was not able to retrieve logs where you mentioned, this may be problem with my 2 computer setup. @Eireen2011  Thanks for sharing your logs instead.

**Comment by antalest** [comment]
@rainmanwy 
Hello!

> I have updated the code, you may get the latest code from github.

How to install your latest code?
I installed Sikuli like this:
`pip install robotframework-SikuliLibrary`
And that gave me version 1.0.1 where this Paste Text error exists.

I tried the following command in windows command prompt too:
`pip install git+https://github.com/rainmanwy/robotframework-SikuliLibrary.git`
But that gave error: Command "python setup.py egg_info" failed with error code 1 in c:\users\ala\appdata\local\temp\pip-req-build-4ycl8p\

Sorry if this is a noob question. I just started learning these things.
